frontends/web/src/routes/device/bitbox02/checkbackup.tsx

```diff
@@ -81,38 +81,34 @@ class Check extends Component<Props, State> {
         >
           {t('button.check')}
         </Button>
-        {
-          activeDialog && (
-            <Dialog title={message}>
-              {
-                <div className="columnsContainer half">
-                  <div className="columns">
-                    <div className="column">
-                      {
-                        foundBackup !== undefined && (
-                          <BackupsListItem
-                            backup={foundBackup}
-                            handleChange={() => undefined}
-                            onFocus={() => undefined}
-                            radio={false} />
-                        )
-                      }
-                    </div>
-                  </div>
-                  <DialogButtons>
-                    <Button
-                      primary
-                      onClick={this.abort}
-                      disabled={!userVerified}
-                    >
-                      { userVerified ? t('button.ok') : t('accountInfo.verify') }
-                    </Button>
-                  </DialogButtons>
+        <Dialog open={activeDialog} title={message}>
+          {
```

Comment:
since you are here, could you remove this `{`, seems to be some leftover that is not needed.

Reply:
Yeah, good catch!